portfolio/src/main/java/com/google/sps/servlets/CommentForumServlet.java

```diff
@@ -109,4 +101,27 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  private int getMaxNumComments(HttpServletRequest request) {
+    String maxNumCommentsString = request.getParameter("max-num-comments");
+    // Convert the input to an int.
+    int maxNumComments;
+    try {
+      maxNumComments = Integer.parseInt(maxNumCommentsString);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + maxNumCommentsString);
+      // If conversion fails, set default value
+      maxNumComments = 5;
+    }
+
+    // Enforce bounds on number of comments displayed
+    if (maxNumComments < 0) {
+      maxNumComments = 0;
+    }
+    if (maxNumComments > 100) {
```

Comment:
nit: Pull the `100` out as a const at the top of the file (like the query object), with a descriptive name. As a rule of thumb, if you need a value more than once, or if it's unclear what it does, it should be a named const.